doc/languages.rst

```diff
@@ -900,6 +900,17 @@ to view the docstring of the syntax checker.  Likewise, you may use
 
       .. syntax-checker-config-file:: flycheck-flake8rc
 
+   .. syntax-checker:: python-mypy
+
+      Type check python with  `mypy <http://www.mypy-lang.org/>`_.
+
+      .. syntax-checker-config-file:: flycheck-python-mypy-ini
+
+      .. defcustom:: flycheck-python-mypy-disable-incremental
+
+         Switches off incremental type checking. This is slower, but
+         prevents mypy from writing ``.mypy_cache`` files.
```

Comment:
I'd suggest to say `creating .mypy_cache folder` instead

Reply:
I also wonder if the setting should be called `enable`, and be off by default.  These folders are really annoying. Opinions?

Reply:
Or maybe there's a way to make mypy put all these folders in one place on the file system, and we should let users customize that?

Reply:
I've updated the doc as you suggested.

I did think about having an enable flag, but this is consistent with the mypy behaviour, which enables caching by default. Personally I don't mind the .mypy_cache directory being created, and I find mypy rather slow otherwise.